tests/garage/torch/algos/test_vpg.py

```diff
@@ -0,0 +1,104 @@
+import random
+
+import gym
+import torch
+
+from garage.envs.base import GarageEnv
+from garage.experiment import deterministic
+from garage.experiment import LocalRunner
+from garage.np.baselines import LinearFeatureBaseline
+from garage.torch.algos import VPG
+from garage.torch.policies import GaussianMLPPolicy
+from tests.fixtures import snapshot_config
+
+
+class TestVPG:
+
+    def test_vpg_no_entropy(self):
+        """Test VPG with no_entropy."""
+        deterministic.set_seed(random.sample(range(100), 1)[0])
```

Comment:
please do not use any random data in tests, especially seeds. the whole point of setting the seed is so that the tests are deterministic.

Reply:
fixed